lib/view_component/test_helpers.rb

```diff
@@ -252,15 +252,18 @@ def __vc_test_helpers_build_controller(klass)
 
     def __vc_test_helpers_preview_class
       result = if respond_to?(:described_class)
+        # :nocov:
         raise "`render_preview` expected a described_class, but it is nil." if described_class.nil?
 
         "#{described_class}Preview"
+        # :nocov:
       else
         self.class.name.gsub("Test", "Preview")
       end
       result = result.constantize
     rescue NameError
       raise NameError, "`render_preview` expected to find #{result}, but it does not exist."
     end
+    # :nocov:
```

Comment:
Why were these changes necessary?

Reply:
I'll try dropping them, but runs were failing due to coverage errors. I think this was the cause, though I'm not sure if anything in this PR in particular would've triggered it.